tests/foreman/destructive/test_clone.py

```diff
@@ -50,7 +50,7 @@ def test_positive_clone_backup(target_sat, sat_ready_rhel, backup_type, skip_pul
     :customerscenario: true
     """
     rhel_version = sat_ready_rhel._v_major
-    sat_version = target_sat.version
+    sat_version = settings.server.version.release
```

Comment:
We do've a `is_stream` property as well to handle this correctly and I think it'd be better to rely on satellite rpm instead of this setting.
```suggestion
    sat_version = 'stream' if target_sat.is_stream else target_sat.version
```

Reply:
Ok awesome lets try that.